### PR TITLE
Post a GitHub deployment instead

### PR DESCRIPTION
**Pull request by lukebjerring**
## Description
Tweaks the behaviour of the deployment messages to use the GitHub deployments API.

### CODE REVIEW

**Comment by lukebjerring** [comment]
@foolip you can see that it "works locally" from the eventual output here https://github.com/web-platform-tests/wpt.fyi/pull/1442

File: util/deploy-comment.sh [hunk 1]
```diff
@@ -20,35 +20,51 @@ if [[ -z "${TRAVIS_REPO_SLUG}" ]];
...
-          "https://api.github.com/repos/${TRAVIS_REPO_SLUG}/issues/${TRAVIS_PULL_REQUEST}/comments"
-else
-    info "Found existing comment mentioning link:\n${STAGING_URL}"
+    debug "Created deployment ${DEPLOYMENT_ID}"
 fi
+
+debug "Setting status to deployed"
+POST_BODY="{
+                \"state\": \"success\",
+                \"target_url\": \"${STAGING_URL}\",
```
Comment (foolip) [inline util/deploy-comment.sh]: This should be `environment_url` per https://developer.github.com/v3/repos/deployments/#update-a-deployment. `target_url` was renamed `log_url` it seems.

I doubt it's useful, but a log URL would be the running Travis job.

**Comment by foolip** [comment]
Eventually we might want to include deployments cleanup as follow up to this, telling GitHub that a deployment is gone.

File: util/deploy-comment.sh (hunk 1 above)
Comment (lukebjerring) [reply util/deploy-comment.sh]:
```suggestion
                \"environment_url\": \"${STAGING_URL}\",
```

**Pull request merged**